video/out/vo_gpu_next.c

```diff
@@ -2058,8 +2058,14 @@ static void update_lut(struct priv *p, struct user_lut *lut)
     // Load LUT file
     char *fname = mp_get_user_path(NULL, p->global, lut->path);
     MP_VERBOSE(p, "Loading custom LUT '%s'\n", fname);
-    struct bstr lutdata = stream_read_file(fname, p, p->global, 100000000); // 100 MB
-    lut->lut = pl_lut_parse_cube(p->pllog, lutdata.start, lutdata.len);
+    const int lut_max_size = 600000000; // 600 MB
```

Comment:
```suggestion
    const int lut_max_size = 1536 << 20; // 1.5 GB
```
We use this value already for the lut cache to support 512x512x512 which apparently is allowed value in mpv. And some additional size for smaller luts.

I think it should be fine to have such limit here too.